diff -ur '--exclude=toolchains' '--exclude=*git*' mingw-builds-orig/library/config.sh mingw-builds/library/config.sh
--- mingw-builds-orig/library/config.sh	2018-12-29 00:11:19.222621800 -0600
+++ mingw-builds/library/config.sh	2018-12-28 19:06:09.155557900 -0600
@@ -36,7 +36,7 @@
 # **************************************************************************
 
 MINGW_W64_BUILDS_VERSION="MinGW-W64-builds-4.3.4"
-MINGW_W64_PKG_STRING="Built by MinGW-W64 project"
+MINGW_W64_PKG_STRING="Built by arkay7777 using scripts from mingw-builds"
 
 # **************************************************************************
 
diff -ur '--exclude=toolchains' '--exclude=*git*' mingw-builds-orig/scripts/tests.sh mingw-builds/scripts/tests.sh
--- mingw-builds-orig/scripts/tests.sh	2018-12-29 00:11:19.866897800 -0600
+++ mingw-builds/scripts/tests.sh	2018-12-28 15:45:40.914232900 -0600
@@ -49,11 +49,11 @@
 	"dll_test2.cpp -L. -ldll2 -o dll_test2.exe"
 )
 
-lto_test_list=(
-	"lto_int.cpp -I$TESTS_DIR -flto -c -o lto_int.o"
-	"lto_test.cpp -I$TESTS_DIR -flto -c -o lto_test.o"
-	"lto_int.o lto_test.o -flto -o lto_test.exe"
-)
+#lto_test_list=(
+#	"lto_int.cpp -I$TESTS_DIR -flto -c -o lto_int.o"
+#	"lto_test.cpp -I$TESTS_DIR -flto -c -o lto_test.o"
+#	"lto_int.o lto_test.o -flto -o lto_test.exe"
+#)
 
 omp_test_list=(
 	"omp_test.c -fopenmp -o omp_test.exe"
